mathics/core/expression.py

```diff
@@ -998,6 +998,7 @@ def rewrite_apply_eval_step(self, evaluation) -> typing.Tuple["Expression", bool
         # @timeit
         def eval_elements():
             nonlocal recompute_properties
+
```

Comment:
I think we are having blank wars here. 

I think we needs to set 

```
trim_trailing_whitespace: “true
``` 

in `.editorconfig`.

Reply:
In any case, if I remove the blank line, flake8 complains about it.